benchopt/base.py

```diff
@@ -48,6 +50,21 @@ class BaseSolver(ParametrizedNameMixin, DependenciesMixin, ABC):
         strategy='iteration'
     )
 
+    @property
+    def solver_strategy(self):
```

Comment:
As this is for private use, lets use `_solver_strategy`
```suggestion
    def _solver_strategy(self):
```